wrappers/python/nntile/model/llama_decoder.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, x: TensorMoments, attention_layer: LlamaAttention,
         # Init activations and list of layers
         self.mlp = llama_mlp
         layers = [input_norm, attention_layer, post_attn_add, post_attn_norm]
-        layers = layers + llama_mlp.layers + [post_mlp_add]
+        layers = layers + [llama_mlp,] + [post_mlp_add]
```

Comment:
is comma in `[llama_mlp,]` required?

Reply:
`llama_mlp` is not `BaseLayer`, but a `BaseModel`, that is why we get all the layers out of it and extend `layers`. `BaseModel` shall list all `BaseLayers` in `layers`. Object `layers` can not contain `BaseModel` inside, as there will be an error when we call some `BaseLayer`-specific function like listing all temporary tensors of a layer. We have to rework it some time later into a single class like `torch.nn.module`.

Reply:
`BaseModel shall list all BaseLayers in layers`
So why did you do separate classes for mlp and not just 'make it all here'? Perhaps because of encapsulation
And then you destroy all your encapsulation just to list layers

Maybe we need to refactor BaseModel to consider some layers can be BaseModel class. What do you think?

Reply:
This design is bad, it shall be done in an other way. There was some logic behind splitting `BaseLayer` and `BaseModel`. `BaseLayer` is about tackling unique non-shared temporary tensors inside it, while `BaseModel` is about activations, which might be shared across some layers.

Reply:
ok, lets use self.decoders implicitly